src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -76,11 +75,13 @@ const useResolveStep = () => {
     currentLocation,
     reasonCodes,
     users,
+    currentUser,
   } = useSelector((state) => ({
     users: getUsers(state),
     cycleCountIds: getCycleCountsIds(state),
     reasonCodes: getReasonCodes(state),
     currentLocation: getCurrentLocation(state),
+    currentUser: state.session.user,
```

Comment:
not a must have, but maybe it'd be good to standardize and create a selector for that, as for the fields above